adapters/huaweiads/huaweiads_req_resp.go

```diff
@@ -0,0 +1,199 @@
+package huaweiads
+
+type HuaweiAdsRequest struct {
```

Comment:
Why are these exported? other files in the same package can reference unexported objects.

Reply:
Deleted, sorry, I often write like this when writing  java

Reply:
I see you moved these all over to the main `huaweiads.go` file, but you are continuing to export them. I think it was a bit more organized to have them over in this file, I was suggesting that you move them. `huaweiads.go` is long enough as it is. My point was you could have gone with `type huaweiAdsRequest struct {` so that Go would know that the struct is only intended to be referenced within this package.

Reply:
already corrected ， thanks